### ISSUE TITLE
Lightning channel force closed with no attempted tx

### ISSUE THREAD

**Issue by bedair81** [MutinyWallet/mutiny-web#1062]
Fresh wallet started up few days ago, loaded some sats through lightning and setup a lighting address through fedimint, all sats were on the lightning channel with nothing on the federation or on chain, logs and screenshots are attached.
![signal-2024-04-15-163309_002](https://github.com/MutinyWallet/mutiny-web/assets/114684096/02345248-65dc-4607-b1a0-5c718c82481f)
![signal-2024-04-15-163309_003](https://github.com/MutinyWallet/mutiny-web/assets/114684096/e0a2cca1-d4e8-4a0c-92a5-f8aa0213508a)
![signal-2024-04-15-163309_004](https://github.com/MutinyWallet/mutiny-web/assets/114684096/6227a2eb-44da-45f5-b629-6f504cd44261)
[mutiny-logs.txt](https://github.com/MutinyWallet/mutiny-web/files/14983653/mutiny-logs.txt)

**Comment by benthecarman** [comment]
Here is the force close transaction: https://mempool.space/tx/355005c0364be2c1f38edb48d09e9c231a41c06f33cae9ca9520fb50b734d00b

Once it gets 6 confirmations it will sweep to your on-chain wallet.

As for the reason why

```
a200 2024-04-15 13:31:33.592 ERROR [lightning::ln::channelmanager:10307] A ChannelManager is stale compared to the current ChannelMonitor!
a200 2024-04-15 13:31:33.592 ERROR [lightning::ln::channelmanager:10308]  The channel will be force-closed and the latest commitment transaction from the ChannelMonitor broadcast.
a200 2024-04-15 13:31:33.592 ERROR [lightning::ln::channelmanager:10310]  The ChannelMonitor for channel 80b0aff5aae10e3026ae7abb584f320fcdf1a58d572603fab5682ac6b8dcecc0 is at update_id 28 but the ChannelManager is at update_id 27.
```

This is a pretty rare storage issue. Seems like the last time it was running, it didn't properly save to storage, causing a force close. Sorry about this

**Comment by benthecarman** [comment]
Let us know if there's anything more that we can do

**Comment by bedair81** [comment]
Appreciate the quick response time! as soon as I saw the channel was closed I knew not much can be done, just wanted to send logs etc to see if the issue can be fixed in future :)

Also the wallet says "you closed a lightning channel" even when the channel was force closed without the users consent, as seen here. It would likely be better if a more detailed message with an explanation as to what a force close is was displayed, and potential reasons why it happened. The wallet is my recently onboarded younger brother and he was very worried he did something wrong until I checked it out. Just my two sats, appreciate you guys!
